Midround Event changes

Cw3040 opened the pull request:
Changelogs in the Commits my friends

Cw3040 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

Cw3040 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

Cw3040 committed:
Random Event Changes v.1 (meteor NOT included)

-Added Devil into Random Events, weight = 1; min_players = 40; max 1
devil/round
-Reduced Pyroclastic Anomaly weight from 20 to 15 and added a minimum
player count for it
-Immovable rod min_players increased to 25 from 15, and weight reduced
to 7 from 10

Cw3040 committed:
Meteor Changes

This needs its own commit
-Minimum players increased to 30 from 15
-Max occurences reduced to 2 from 3
-Earliest start increased to 45 minutes from 25
-Weight unchanged, but the rate of Catastrophic meteor showers were cut
in half from 10% to 5% chance.

Cw3040 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

deathride58 reviewed (dismissed):
Modularize these changes by overriding them in modular_citadel or add `//CITADEL CHANGE - adjusts x to y` comments to every single line you've changed

deathride58 reviewed (changes requested):
hec wrong button

Cw3040 committed:
Revert "Random Event Changes v.1 (meteor NOT included)"

This reverts commit dc537bd49556d3d19aedb5c0814a5d8b971707cf.

Cw3040 committed:
Revert "Meteor Changes"

This reverts commit 140e02f221d650aed2cd982c704b83cdb1045c51.

Cw3040 committed:
Modularization

These are identical to the prior commits (Random Event Changes & Meteor
Changes) but in modular_citadel/code/modules/events

silicons commented on modular_citadel/code/modules/events/devil.dm:
```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/devil
```
No devil.
Just no.

silicons reviewed (changes requested):
Therei s no possible way you can convince me spawning a devil without a crazy admin doing it is a good idea.

Cw3040 committed:
Remove devil

because Kev doesn't like it so its bad

deathride58 reviewed (changes requested):
Add the new files to the .dme, and make sure the files only override the variables you've changed

Cw3040 committed:
cleaning up and doing shit right

-actually added the new files to the dme
-actually made the files themselves not override everything and cause
issues
-actually wasn't shit at byond for once thanks papa bhijn

Cw3040 committed:
Merge remote-tracking branch 'refs/remotes/Citadel-Station-13/master'

Cw3040 committed:
Feeble Attempt to make Travis happy

commented out the /tg/ vars in \code\...\X.dm that are overridden by the
vars in \modular_citadel\code\...\X.dm
